go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
For compatibility, ensure that you're using compatible versions of dependencies, especially those that are marked with `+incompatible`, which may cause issues in some setups.